codespell_lib/data/dictionary.txt

```diff
@@ -9735,6 +9736,7 @@ boostrappers->bootstrappers
 boostrapping->bootstrapping
 boostraps->bootstraps
 booteek->boutique
+bootime->boottime
```

Comment:
Is that word? Perhaps `boot time` or `boot-time` instead?

Reply:
I made the suggestion two words.